packages/twenty-server/src/engine/core-modules/billing/services/billing-plan.service.ts

```diff
@@ -127,33 +110,21 @@ export class BillingPlanService {
         BillingExceptionCode.BILLING_PLAN_NOT_FOUND,
       );
     }
-    const { baseProduct, meteredProducts, otherLicensedProducts } = plan;
-    const baseProductPrice = baseProduct.billingPrices.find(
-      (price) => price.interval === interval && price.active,
-    );
+    const { meteredProducts, licensedProducts } = plan;
 
-    if (!baseProductPrice) {
-      throw new BillingException(
-        'Base product active price not found for given interval',
-        BillingExceptionCode.BILLING_PRICE_NOT_FOUND,
-      );
-    }
     const filterPricesByInterval = (product: BillingProduct) =>
-      product.billingPrices.filter(
-        (price) => price.interval === interval && price.active,
-      );
+      product.billingPrices.filter((price) => price.interval === interval);
```

Comment:
no more active check?

Reply:
The active check is in the SQL query above.